apps/wallet/src/ui/app/index.tsx

```diff
@@ -139,59 +141,62 @@ const App = () => {
 		return <StorageMigrationPage />;
 	}
 	return (
-		<Routes>
-			<Route path="/welcome" element={<WelcomePage />} />
-			<Route path="forgot-password" element={<ForgotPasswordPage />} />
-			<Route path="restricted" element={<RestrictedPage />} />
-
-			<Route path="/initialize" element={<InitializePage />}>
-				<Route path="select" element={<SelectPage />} />
-				<Route path="create" element={<CreatePage />} />
-				<Route path="import" element={<ImportPage />} />
-				<Route path="backup" element={<BackupPage />} />
-				<Route path="backup-imported" element={<BackupPage mode="imported" />} />
-			</Route>
-
-			<Route path="/*" element={<HomePage />}>
-				<Route path="apps/*" element={<AppsPage />} />
-				<Route path="kiosk" element={<KioskDetailsPage />} />
-				<Route path="nft-details" element={<NFTDetailsPage />} />
-				<Route path="nft-transfer/:nftId" element={<NftTransferPage />} />
-				<Route path="nfts/*" element={<AssetsPage />} />
-				<Route path="onramp" element={<OnrampPage />} />
-				<Route path="receipt" element={<ReceiptPage />} />
-				<Route path="send" element={<TransferCoinPage />} />
-				<Route path="send/select" element={<CoinsSelectorPage />} />
-				<Route path="stake/*" element={<Staking />} />
-				<Route path="tokens/*" element={<TokenDetailsPage />} />
-				<Route path="transactions/:status?" element={<TransactionBlocksPage />} />
-				<Route path="*" element={<Navigate to="/tokens" replace={true} />} />
-			</Route>
-
-			<Route path="accounts/*" element={<AccountsPage />}>
-				<Route path="add-account" element={<AddAccountPage />} />
-				<Route path="add-account" element={<AddAccountPage />} />
-				<Route path="import-ledger-accounts" element={<ImportLedgerAccountsPage />} />
-				<Route path="import-passphrase" element={<ImportPassphrasePage />} />
-				<Route path="import-private-key" element={<ImportPrivateKeyPage />} />
-				<Route path="manage" element={<ManageAccountsPage />} />
-				<Route path="protect-account" element={<ProtectAccountPage />} />
-			</Route>
-
-			<Route path="/account">
+		<NicknamesProvider>
```

Comment:
I've been putting Providers in the `PageLayout` component at `/src/ui/app/pages/layout/index.tsx` - not a big deal but we should probably be consistent in where we wrap the tree with providers to make it easy to identify / refactor in the future